{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -51,6 +51,9 @@
     "django.contrib.contenttypes",
     "django.contrib.sessions",
     "django.contrib.messages",
+    {% if cookiecutter.client_app == "React" %}
+    "whitenoise.runserver_nostatic",
+    {% endif %}
```

Comment:
So this will fail linting because when it generates it'll leave behind blank lines (issue I was talking about)
the hotfix is to have it all on one line.
I assume there is a clear fix though (hopefully)

Reply:
@oudeismetis the fix: https://ttl255.com/jinja2-tutorial-part-3-whitespace-control/